tests/test_cookies.py

```diff
@@ -138,7 +138,7 @@ def handler(request):
     assert response.cookies["test"].value == "pass"
 
 
-@pytest.mark.parametrize("max_age", ["0", 30, "30"])
+@pytest.mark.parametrize("max_age", ["0", 30, "30", "test"])
```

Comment:
Possibly a float value can be thrown into the mix of test parameters?

Reply:
Agreed.